### PR TITLE
docs(root): add accessible labels to chip icons

### PR DESCRIPTION
**Pull request by GCHQ-Developer-299**
## Summary of the changes

Update IcChip guidance and examples to have aria-labels 

## Related issue

#1560

## Checklist

- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

**Review by GCHQ-Developer-530 — APPROVED** [review]

File: src/content/structured/components/chips/code.mdx
Comment (jd239) [inline src/content/structured/components/chips/code.mdx:1]: Could we improve the indentation in our code snippets? I appreciate it's not in this ticket but shouldn't take too long to complete.

Comment (jd239) [reply src/content/structured/components/chips/code.mdx:1]: Example of the tags not lining up correctly:

image

**Review by jd239 — COMMENTED** [review]
Minor update to the code snippets

**Review by GCHQ-Developer-530 — APPROVED** [review]

File: src/content/structured/components/chips/code.mdx [hunk 1]
```diff
@@ -115,28 +121,30 @@ export const snippetsOutline = [
...
-    </svg>
-  </ic-chip>`,
+  <svg
+    slot="icon"
+    viewBox="0 0 24 24"
+    fill="currentColor"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-label="coffee"
+  >
+      <path d="M2,21H20V19H2M20,8H18V5H20M20,3H4V13A4,4 0 0,0 8,17H14A4,4 0 0,0 18,13V10H20A2,2 0 0,0 22,8V5C22,3.89 21.1,3 20,3Z"/>
```
Comment (jd239) [inline src/content/structured/components/chips/code.mdx:131]: This needs to be outdented a bit.

File: src/content/structured/components/chips/code.mdx [hunk 2]
```diff
@@ -115,28 +121,30 @@ export const snippetsOutline = [
...
-    </SlottedSVG>
-  </IcChip>`,
+  <SlottedSVG
+    slot="icon"
+    viewBox="0 0 24 24"
+    fill="currentColor"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-label="coffee"
+  >
+      <path d="M2,21H20V19H2M20,8H18V5H20M20,3H4V13A4,4 0 0,0 8,17H14A4,4 0 0,0 18,13V10H20A2,2 0 0,0 22,8V5C22,3.89 21.1,3 20,3Z"/>
```
Comment (jd239) [inline src/content/structured/components/chips/code.mdx:145]: Needs to be outdented a bit

File: src/content/structured/components/chips/code.mdx [hunk 3]
```diff
@@ -167,32 +176,34 @@ export const snippetsDismissible = [
...
-    chip.addEventListener('icDismiss', (ev) => console.log(ev));
-  </script>`,
+  <svg
+    slot="icon"
+    viewBox="0 0 24 24"
+    fill="currentColor"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-label="coffee"
+  >
+      <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z"/>
```
Comment (jd239) [inline src/content/structured/components/chips/code.mdx:186]: Needs to be outdented a btit. Same position as the React example below

File: src/content/structured/components/chips/code.mdx [hunk 4]
```diff
@@ -238,6 +251,7 @@ export const snippetsDisabled = [
     viewBox="0 0 24 24"
     fill="currentColor"
     xmlns="http://www.w3.org/2000/svg"
+      aria-label="coffee"
```
Comment (jd239) [inline src/content/structured/components/chips/code.mdx:254]: Line up with other attributes

**Review by jd239 — CHANGES_REQUESTED** [review]
Some very very minor updates

**Review by jd239 — APPROVED** [review]

**Review by zxh722 — APPROVED** [review]

**Pull request merged**